xla/README.md

```diff
@@ -0,0 +1,124 @@
+# TPU support
+
+This project utilizes [`Fabric`](https://lightning.ai/docs/fabric/stable), which supports TPUs via [PyTorch XLA](https://github.com/pytorch/xla).
+
+To set up a Google Cloud instance with a TPU v4 VM, run the following commands:
```

Comment:
(Optional) Should we include a guide on how to setup gcp? or is this expected as a prereq?